<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeSettingDlg</class>
 <widget class="QDialog" name="TimeSettingDlg">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>181</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>26</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>时间设置</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="monSpin">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>181</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QPushButton" name="timeSet_but">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>时间设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>66</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_36">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>分</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="minSpin">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>66</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
   <property name="value">
    <number>59</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="daySpin">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>181</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>31</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="hourSpin">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>66</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>23</number>
   </property>
   <property name="value">
    <number>23</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="yearSpin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>2000</number>
   </property>
   <property name="maximum">
    <number>2035</number>
   </property>
   <property name="value">
    <number>2011</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="secSpin">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>66</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
   <property name="value">
    <number>59</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>时</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hourBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="secBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yearBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="monBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dayBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_point">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改值:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEdit</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_9">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_del">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>440</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_enter">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>121</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>输入</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_0">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>430</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>yearSpin</tabstop>
  <tabstop>monSpin</tabstop>
  <tabstop>daySpin</tabstop>
  <tabstop>hourSpin</tabstop>
  <tabstop>minSpin</tabstop>
  <tabstop>secSpin</tabstop>
  <tabstop>timeSet_but</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
